spec.md

```diff
@@ -273,7 +273,7 @@ encapsulated within the `data` attribute.
 
 ### data
 * Type: `Any`
-* Description: The event payload. The payload depends on the eventtype and
+* Description: The event payload. The payload depends on the `type` and
   the schemaURL. It is encoded into a media format
```

Comment:
if you use back-ticks on "type" then for consistency you should on "schemaURL" on the next line, and "contenttype" - and we should be consistent on the case: e.g. schemaURL vs contenttype

Reply:
Done